.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -1,22 +0,0 @@
----
-name: Issue template
-about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
----
```

Comment:
Did you mean to delete the template? 🤔

Reply:
Yes, that way it'll inherit from the [global template](https://github.com/swup/.github/tree/master/.github/ISSUE_TEMPLATE) of the swup org.